client/gutenberg/editor/components/post-preview-button/index.jsx

```diff
@@ -127,11 +127,11 @@ export default compose( [
 		const postType = getPostType( getEditedPostAttribute( 'type' ) );
 		const previewLink = getEditedPostPreviewLink();
 
-		const featuredImage = get(
-			getMedia( getEditedPostAttribute( 'featured_media' ) ),
-			'source_url',
-			null
-		);
+		const featuredImageId = getEditedPostAttribute( 'featured_media' );
+
+		const featuredImage = featuredImageId
+			? get( getMedia( getEditedPostAttribute( 'featured_media' ) ), 'source_url', null )
```

Comment:
Nitpick: can't we reuse the `featuredImageId` variable defined a couple of lines above?

Reply:
You Sir is absolutely right and it's not nitpicking. Thank the maker code reviews exist, so, thing like this get caught. Note to self: don't work until 3AM anymore.

Updated on d340cce83dbc6ee63d99e6ba67524fd95f0b6208